docs/core/integration-quality-scale/rules/iqs002.md

```diff
@@ -64,7 +64,8 @@ More information about config flows can be found in the [config flow documentati
 
 ## Exceptions
 
-There are no exceptions to this rule.
+Integrations that don't have a connection to a device or service (for example helpers) don't need to test a connection in the config flow are exempt from this rule.
```

Comment:
```suggestion
Integrations that don't have a connection to a device or service (for example helpers) don't need to test a connection in the config flow and are exempt from this rule.
```

Reply:
"Integrations ... are exempt"

I see you can read this sentence in both ways, hmm

Reply:
No, it was wrong before. My suggestion makes it grammaticaly correct. If you want to make the config flow the subject you need to rewrite the sentence further.